code/libs/ffmpeg/FFmpegContext.cpp

```diff
@@ -100,9 +98,8 @@ std::unique_ptr<FFmpegContext> FFmpegContext::createContext(CFILE* mediaFile) {
 	if (probe_ret < 0) {
 		char errorStr[1024];
 		av_strerror(probe_ret, errorStr, 1024);
-		auto fmt = boost::format("Could not open movie file! Error: %s") % errorStr;
 
-		throw FFmpegException(fmt.str());
+		throw FFmpegException(std::string("Could not open movie file! Error: ") + errorStr);
```

Comment:
Please use `SCP_string` instead of `std::string`. It's just a typedef but we prefer to use it to make changing the string type easier.